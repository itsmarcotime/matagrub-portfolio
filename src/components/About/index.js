import React from "react";
import breenGrub from "../../assets/images/BreenGrub-logo.jpg"

function About () {
    return (
        <section>

            <div className="status-box">
                <h3 className="status-text">
                    Subject: Marco A. Mata <br /><br />
                    Status: Awaiting Assignment
                </h3>
            </div>

        </section>
    );
}

export default About;