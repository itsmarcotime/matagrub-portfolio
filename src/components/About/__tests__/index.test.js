import React from "react";
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import About from "..";

afterEach(cleanup);

describe('about component', () => {
    //baseline render
    it('renders', () => {
        render(<About />);
    });

    //snapshot test
    it('matches snapshot on DOM', () => {
        const { asFragment } = render(<About />);

        expect(asFragment()).toMatchSnapshot();
    });
});
