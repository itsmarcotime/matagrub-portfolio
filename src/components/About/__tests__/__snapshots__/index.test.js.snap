// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about component matches snapshot on DOM 1`] = `
<DocumentFragment>
  <section>
    <div
      class="d-flex aboutContainer"
    >
      <div
        class="col-4 text-center statusBox"
      >
        <p
          class="aboutStatus"
        >
          SUBJECT: MARCO A. MATA
          <br />
          <br />
          STATUS: OBSERVATION 
          <br />
          <br />
          REASON:
          <br />
           AWAITING FURTHER DATA
        </p>
      </div>
      <div
        class="col-4"
      >
        <img
          alt="portfolio-pic"
          class="aboutImg"
          height="300"
          src="portfolioPic.jfif"
          width="300"
        />
      </div>
      <div
        class="col-4 text-center statusBox"
      >
        <p
          class="aboutText"
        >
          DESCRIPTION:
        </p>
        <p
          class="aboutText"
        >
          Subject has completed a full stack programming bootcamp from University of Texas in San Antonio. Strong interests in understanding the deeper meaning and functions of everything around us. Continuous learner who loves working with JavaScript, React, and Python. Born to be a back-end developer, but will never be afraid of front-end work. Hobbies include jogging, philosophy, physics, and eating Pho. üçú
        </p>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
